src/Util/DatasetCitationUtil.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Util;
+
+use App\Entity\Dataset;
+
+/**
+ * A utility to create a Dataset Citation Text.
+ */
+class DatasetCitationUtil
+{
+    /**
+     * Creates the and return the Dataset Citation Text.
+     *
+     * @param Dataset $dataset The Dataset that has the data.
+     *
+     * @return string The Dataset Citation string.
+     */
+    public static function getCitation(Dataset $dataset) :string
+    {
+        $title = $dataset->getTitle();
+        $title = preg_replace('/\.$/', '', $title);
```

Comment:
Do we really have titles that are just a period? If so, why should we be cleaning it up here?

Reply:
This is how it was, it removes the last period, we may want to remove this.